package-lock.json

```diff
@@ -10,7 +10,6 @@
       "hasInstallScript": true,
       "license": "ISC",
       "workspaces": [
-        "docs",
```

Comment:
issue: I think the one in `package.json` needs to be removed also?

Reply:
gah i cherrypicked. let me reverse this change first, my upstream PR should fix

Reply:
rebased